packages/meta-router/src/lib/meta-router.ts

```diff
@@ -168,6 +168,7 @@ export class MetaRouter implements Router {
 		options?: Partial<{
 			permitOptions: Omit<StandardPermitArguments, 'amount'>;
 			zeroExOptions: Partial<Zero0xQuoteOptions>;
+			maxSlippage: BigNumber;
```

Comment:
In the contracts this value is maxLoss not maxSlippage, if you agreed with frontend to pass the maxSlippage here it should be converted to maxLoss and checked if it is within acceptable range. Example:
If they pass `500`(0.5%) we should make a conversion into `9500`.